doc/classes/@GlobalScope.xml

```diff
@@ -1196,6 +1196,29 @@
 				else:
 				    print("Unexpected result")
 				[/codeblock]
+				See also [method typeof_string]
+			</description>
+		</method>
+		<method name="typeof_string">
+			<return type="String" />
+			<param index="0" name="type" type="int" />
+			<description>
+				Returns a human-readable name of the given [param type], using the [enum Variant.Type] values.
+				[codeblock]
+				var values = [1, 2.5, "Hello", true]
+				
+				for value in values:
+					var type_id = typeof(value)
+					var type_name = typeof_string(type_id)
+					print("%s is %s" % [value, type_name])
```

Comment:
You don't need string formatting or looping through an array to explain this concept. Keep it as simple as possible.

Reply:
I specifically did it this way to avoid this function nesting, that might look intimidating at first.
```suggestion
				print(typeof_string(typeof(values[0]))) # Prints int
				print(typeof_string(typeof(values[1]))) # Prints float
				print(typeof_string(typeof(values[1]))) # Prints String
				print(typeof_string(typeof(values[3]))) # Prints bool
```
The solution might be to use more variables, but this would make the code block a lot longer and harder to follow
```suggestion
				var a = 1
				var b = "Hello"
				var c = true
				
				var a_type = typeof(a)
				var b_type = typeof(b)
				var c_type = typeof(c)
				
				print(typeof_string(a_type)) # Prints int
				print(typeof_string(b_type)) # Prints String
				print(typeof_string(c_type)) # Print bool
```

Reply:
Yes, this is better. 2 variables suffice, really.  Also, since the method returns a string, wrap it with quotation marks inside the comments, like `# Prints "String"`

Reply:
What do you think about this?

```suggestion
				func _ready():
					add_one(1) # Prints 2
					add_one(3.0) # Prints 4
					add_one("7") # Prints "Can't add one to String"
					
				func add_one(x):
			                var type = typeof(x)
					if type == TYPE_INT or type == TYPE_FLOAT:
					        print(x + 1)
					else:
					        print("Can't add one to ", typeof_string(type))
				
```

Reply:
Too complicated. I just want to know how `typeof_string()` works, this example forces me to think about irrelevant things, which is massively discouraged in documentation. How about just this?

```
typeof_string(TYPE_INT)  # Prints "int"
typeof_string(TYPE_STRING)  # Prints "String"
```

Reply:
And what about this?

```gdscript
print(TYPE_INT) # Prints 2
print(typeof_string(TYPE_INT)) # Prints "int"
print(typeof_string(TYPE_STRING))  # Prints "String"
```

Reply:
Oop I was referring to the old one! Well, we practically reached the same endpoint then.

Don't forget to amend your commits.

Reply:
Thanks for the feedback ❤️